src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<UserControl x:Class="Dynamo.DocumentationBrowser.DocumentationBrowserView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:p="clr-namespace:Dynamo.Wpf.Properties;assembly=DynamoCoreWpf"
+             xmlns:fa="http://schemas.fontawesome.io/icons/"
+             xmlns:w="clr-namespace:Dynamo.DocumentationBrowser.Properties"
+             xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        
+            xmlns:WPF="clr-namespace:Microsoft.Toolkit.Wpf.UI.Controls;assembly=Microsoft.Toolkit.Wpf.UI.Controls.WebView"
```

Comment:
can you verify you tested this in Revit? My experience is that this control does not work in Civil3d Dynamo, I'm guessing Revit is similar, but unfortunately we need something that works everywhere, even if it does work in Revit.

I have found that MSwebBrower in `system.windows.controls` works well and can be forced to display in IE11 mode which has pretty good support - it displays librariejs without issue so it should display these static html pages fine. 

In any case, I understand you don't have time to change this up which is fine, I am just wondering if you tested it in Revit or Civil at this point.

Reply:
I've now switched the browser to use the `System.Windows.Controls.WebBrowser` control, @shknudsen will also be testing with `DynamoRevit` tomorrow.

Reply:
@mjkkirschner @radumg 

![test](https://user-images.githubusercontent.com/13732445/73860503-90514a80-4833-11ea-82f4-f05288801156.gif)

Everything seems to be working as expected on D4R, except for the white screen bug that happens the first time you open the extension.